pkg/client/reputation.go

```diff
@@ -87,11 +87,20 @@ func (c *clientImpl) SendLocalTrust(ctx context.Context, prm SendLocalTrustPrm,
 
 // SendIntermediateTrustPrm groups parameters of SendIntermediateTrust operation.
 type SendIntermediateTrustPrm struct {
-	iter uint32
+	epoch uint64
```

Comment:
Can you add empty line to keep all `*Prm` structures in one style?

Reply:
added